### ISSUE TITLE
21k 30k 58.5k  wrong ？

### ISSUE THREAD

**Issue by lsq357** [r9y9/deepvoice3_pytorch#17]
Pretrained models of DeepVoice3 Only need 21k steps to train ?
In my experiment.I think 21k steps is too small to train. 
Maybe you write 210k to  21k?
And 30k for Nyanko, 58.5k for multi-deepvoice3?

**Comment by r9y9** [comment]
> Pretrained models of DeepVoice3 Only need 21k steps to train ?
> In my experiment.I think 21k steps is too small to train.

Sorry for the confusion. Honestly, I don't remember correctly as I noted at https://r9y9.github.io/deepvoice3_pytorch/#fn:1. The checkpoint name says it's trained 21k steps. However, I might have trained 21k steps futher from a pretrained model. Also, there was a bug (https://github.com/r9y9/deepvoice3_pytorch/issues/1#issuecomment-352441834) that dropout was unexpectedly disabled while I was working on single-speaker DeepVoice3. It might lead to faster convergence (and overfitting).

> And 30k for Nyanko, 58.5k for multi-deepvoice3?

58.5k for Nyanko, 60k (30k + 30k) for multi-deepvoice3.

**Comment by r9y9** [comment]
Oops, sorry, I was too stupid. 210k, not 21k. I will fix them as soon as possible..

**Comment by r9y9** [comment]
Fixed at https://github.com/r9y9/deepvoice3_pytorch/commit/e513a76b395af31b3436c784ae526e976e3566db
